src/components/HTMLEngineProvider/HTMLRenderers/VideoRenderer.tsx

```diff
@@ -36,7 +34,7 @@ function VideoRenderer({tnode, key}: VideoRendererProps) {
                         <VideoPlayerPreview
                             key={key}
                             videoUrl={sourceURL}
-                            reportID={currentReportIDValue?.currentReportID ?? '-1'}
+                            reportID={report?.reportID ?? '-1'}
```

Comment:
@suneox We cannot remove the default value of reportID here, because if we do that we cannot play the video in the search page.